src/lib/routes/admin-api/config.ts

```diff
@@ -55,7 +55,7 @@ class ConfigController extends Controller {
             permission: NONE,
             middleware: [
                 openApiService.validPath({
-                    tags: ['admin'],
+                    tags: ['Admin UI'],
```

Comment:
I wonder how the space and capitalization affects the client API generator, which uses tag names in class names. Hopefully it just works. (Like `AdminApi` from the current `admin` tag.)

Reply:
Yeah, generators do support it (at least the ones I've tried), but I'd be happy to make it admin-ui or something instead 💁🏼